src/log-store/src/raft_engine/backend.rs

```diff
@@ -137,29 +139,48 @@ impl KvBackend for RaftEngineBackend {
 
     async fn range(&self, req: RangeRequest) -> Result<RangeResponse, Self::Error> {
         let mut res = vec![];
+        let (start, end) = req.range();
+        let RangeRequest {
+            keys_only, limit, ..
+        } = req;
+
+        let (start_key, end_key) = match (start, end) {
+            (Included(start), Included(_)) => {
+                (Some(start.clone()), Some(get_next_prefix_key(&start)))
```

Comment:
```suggestion
                (Some(start), Some(end))
```

Reply:
The raft engine use the [start,end) pattern.